app/views/api/v2/smart_proxies/main.json.rabl

```diff
@@ -4,6 +4,10 @@ extends "api/v2/smart_proxies/base"
 
 attributes :created_at, :updated_at
 
+node :version do |proxy|
```

Comment:
I'm a little unsure about including this in the main rabl, since it'll cause a REST query and I don't think there's any precedent for it in the API currently.  I think it's best to leave only the POST /version route already added and remove it from here.